DSNP-Wallet/Source/Common/Foundation/ViewControllerFactory.swift

```diff
@@ -54,7 +56,12 @@ enum ViewControllerFactory: String, CaseIterable {
             let storyboard = UIStoryboard(name: storyboardName, bundle: nil)
             vc = storyboard.instantiateInitialViewController()
         } else {
-            vc = UIViewController.fromString(viewControllerName: self.className)
+            // TODO: Temp Fix
+            if self.className == "QaViewController" {
```

Comment:
Here, we have a slightly hacky solution. The reason being that `UIViewController.fromString(viewControllerName: )` initializes a view controller using `init(nibName: , bundle:)` behind the scenes. We get a crash telling us this. The reason is `QAViewController` doesn't subclass `UIViewController`. Since we don't have our final UI implemented, I figure we can leave this as is because this will all change anyway